### PR TITLE
update readme, update angular fire to 5.2.3

### PR DESCRIPTION
**Pull request by kawyua**
scrreadme
added circleci build passing status, website link, and documentation section/link

Edit:
removed tutorial folder and readme.txt and dependency to compodoc
update angular fire to 5.2.3. This version says it works with windows and supports firebase 7 =)
https://github.com/angular/angularfire/releases
changed command to ` npx compodoc -s` to run locally

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
+[![CircleCI](https://circleci.com/gh/CSC59939/Chitty-Chat.svg?style=svg)](https://circleci.com/gh/CSC59939/Chitty-Chat)
 # ChittyChat
-Open the chitty-chat website and chat to each other with an emoji analysis!
+Open http://chat.chitty-chat.com/ and chat to each other with an emoji analysis!
```
Comment (angelica-hernandez) [inline README.md]: We can also use chittychat.space, should also be put in the readme too

Comment (kawyua) [reply README.md]: We can link with https now so i changed it to https://chat.chitty-chat.com/ OR https://chittychat.space/
done!

**Comment by kawyua** [comment]
> 
> 
> i had to install compodoc globally with `npm install -g @compodoc/compodoc`. before running with `compodoc -s`
> so, just `npm install` might not be enough.
> maybe there's a fix for that?

`npx compodoc -s` should allow us to run compodoc locally without installing globally. done!

**Comment by kawyua** [comment]
requesting re-review from @FarhanHyder !

**Pull request merged**